### PR TITLE
[PD-3205] Enforce Activities on CBO API

### PR DESCRIPTION
**Pull request by koonsies**
## What's this PR do?

Gets the CBO activities in line with activities from other apps and enforces their use in the API calls.

## Where should the reviewer start?

Manage Users
* Click 'Roles'
* Create a new role
* Assign all of the manage users activities to that role.
* Assign your user account to that role
* Click 'Users' on the side navigation.
* Edit your User account
* Remove "admin" from your roles
* Add/Remove CBO permissions to test what happens when they aren't present.

When finished, edit your user and readd the Admin role.

## How should this be manually tested?

Tediously.

## Any background context you want to provide?

None that I want to, no.

### CODE REVIEW

File: myjobs/models.py [hunk 1]
```diff
@@ -1363,6 +1363,12 @@ def end(sender, impersonator, impersonating, request, **kwargs):
 
 
 ACTIVITIES = {
+    'CBO' : [
+        ("create cbo", "Create data for CBO app, including cbo, contact, etc"),
```
Comment (aljordan82) [inline myjobs/models.py]: Is the "CBO" in the description visible to users?

Comment (koonsies) [inline myjobs/models.py]: Yeah, it's visible in Manage Users. We haven't settled on a replacement for CBO, officially, and when we do we'll have to do a big refactor to fix this, the urls, and other places it says "CBO".

Comment (tdphillips) [inline myjobs/models.py]: I think we may have had the replacement chosen for us already.

https://github.com/DirectEmployers/MyJobs/pull/2979

Comment (koonsies) [inline myjobs/models.py]: Ahhh. Didn't realize the refactor had gone through already. Fair enough.

**Pull request closed without merging**